framework/src/media/HttpStream.cpp

```diff
@@ -0,0 +1,184 @@
+/******************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+#include <curl/curl.h>
+#include <curl/easy.h>
+#include <debug.h>
+
+#include "HttpStream.h"
+
+namespace media {
+namespace stream {
+
+#define SET_OPTION(curl, opt, val) \
+	do {\
+		auto result = curl_easy_setopt(curl, opt, val);\
+		if (result != CURLE_OK) {\
+			meddbg("curl_easy_setopt(%d...) failed!\n", (int)opt);\
+			return false;\
+		}\
+	} while (0)
+
+int HttpStream::mInitialzeCount = 0;
```

Comment:
please move to header file. let it use as a private variable

Reply:
It's already a private variable, but static, so we need initialize it here.